src/devices/CharacterLcd/LcdInterface.I2c.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.Device.I2c;
+using System.Diagnostics;
+
+namespace Iot.Device.CharacterLcd
+{
+    public abstract partial class LcdInterface : IDisposable
```

Comment:
I think something like this (minus LCD specific stuff) would be useful in many other devices - we should look closely for similar patterns

Reply:
I have been looking for patterns. So far I'm not finding enough commonality to warrant pulling anything up.